api/v1alpha1/envoyproxy_metric_types.go

```diff
@@ -60,7 +60,7 @@ type ProxyPrometheusProvider struct {
 }
 
 // Match defines the stats match configuration.
-type Match struct {
+type Match struct { // TODO: zhaohuabing this type should be renamed to StatsMatch
```

Comment:
can you open an issue to track this?

Reply:
https://github.com/envoyproxy/gateway/issues/2067